python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_functional_chaos.py

```diff
@@ -14,10 +14,10 @@
 # In this example we create a global approximation of a model response using
 # polynomial chaos expansion.
 #
-# Let :math:`h` be the function defined by:
+# Let :math:`\vect{g}` be the function defined by:
 #
 # .. math::
-#    g(\mathbf{x}) = \left[\cos(x_1 + x_2), (x_2 + 1) e^{x_1}\right]
+#    \vect{g}(\mathbf{x}) = \left[\cos(x_1 + x_2), (x_2 + 1) e^{x_1}\right]
 #
 #
 # for any :math:`\mathbf{x}\in\mathbb{R}^2`.
```

Comment:
I suggest to search for the `mathbf` macro and to see if the `vect` macro is appropriate as a replacement.
```suggestion
# for any :math:`\vect{x} \in \Rset^2`.
```